source/fe/fe_system.cc

```diff
@@ -2011,7 +2011,7 @@ FESystem<dim, spacedim>::initialize(
           // This logic isn't valid for higher-order wedges because the first
           // two faces are triangles, so no table will be set up for
           // quadrilateral faces
-          Assert(this->n_dofs_per_quad(face_no) == 0 ||
+          Assert(this->degree <= 2 ||
```

Comment:
@drwells Could you take a look here? This assert is new to me...

Reply:
👍 I added this in #14348. At the time, I misunderstood what `n_unique_quads()` returned - I thought it would be `2` for wedges but it is `5` so that assertion (and its text) are not correct.